8_viz.yml

```diff
@@ -27,7 +27,8 @@ targets:
       temp_data_ind = '7b_temp_merge/out/merged_temp_data_daily.feather.ind',
       lake_names_ind = '2_crosswalk_munge/out/lake_names_crosswalk.rds.ind',
       dow_cross_ind = '1_crosswalk_fetch/out/nhd2dowlknum.RData.ind',
-      state_cross_ind = '4_params_munge/out/nhd2states.rds.ind')
+      state_cross_ind = '4_params_munge/out/nhd2states.rds.ind',
+      MN_bathy_ind = '4_params_munge/out/MN_bathy.rds.ind')
 
   8_viz/inout/lakes_summary.feather:
     command: gd_get('8_viz/inout/lakes_summary.feather.ind')
```

Comment:
Can't comment on the file, so doing it here. If we are using an .ind, we shouldn't be checking the map into the repo (should gitignore it). For 8_viz/out/lakes_summary_fig.html

Reply:
Oops, this was my bad! Kelsey, can you add `html` to the `gitignore`?

Reply:
Need to do one more step too: https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore

Reply:
I think I took care of it -- let me know if there's anything else!

Reply:
Looks good. I think sam is on vacation but when she's back, I will defer to her for :+1: :-1: on merge